### ISSUE TITLE
Issue initializing tile layer

### ISSUE THREAD

**Issue by burakulutoprak** [gwidgets/gwty-leaflet#6]
Hi, 
With the latest snapshot when i try to bootstrap the map i see only gray background. Then I noticed in the leaflet quick start tutorials I need to initialize tile layer which than gives javascript error saying id is not defined. 
Is there any startup project that can show an actual map without gray area, for the latest leaflet version?
Thanks. 

`		String tileUrl = "https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token=XXX";
		LatLng center = L.latLng(41.130076, 29.256913);
		Map map = L.map("map", new MapOptions.Builder(center,13,5).build());
		L.tileLayer(tileUrl, new TileLayerOptions.Builder().build()).addTo(map);
`

**Comment by zak905** [comment]
I have updated the gwty-leaflet-starter-guide to use version 0.5-SNAPSHOT. 104d5ce2433f803fe1beccdf86594e61724e98ef
If you see only gray background than the the coordinates or the zoom should be wrong. Do you have your project somewhere in github?

**Comment by burakulutoprak** [comment]
I tried the starter project and it works fine.  I am not sure why i could not make it work , but the difference i was trying was to use the latest leaflet javascript files. I will try again if it works fine. 

At first I thought map is not working properly because dragging and zooming was disabled. I suggest to enable them. I changed following line in the starter which makes the demo more intuitive. 
MapOptions options = new MapOptions.Builder(L.latLng(52.51, 13.40), 12, 5).maxZoom(13).dragging(true).build();

Is there Google tiles integration with leaflet google plugin. 
Thanks alot. 
Great work.
